src/main/kotlin/graphics/scenery/Icosphere.kt

```diff
@@ -177,14 +189,10 @@ open class Icosphere(val radius: Float, val subdivisions: Int) : Mesh("Icosphere
                     }
                 }
 
-                uv1.xy().get(texcoords).position(texcoords.position() + 2)
-                uv2.xy().get(texcoords).position(texcoords.position() + 2)
-                uv3.xy().get(texcoords).position(texcoords.position() + 2)
+                pTxc++[0] = uv1.xy()
+                pTxc++[0] = uv2.xy()
+                pTxc++[0] = uv3.xy()
             }
-
-            vertices.flip()
-            normals.flip()
-            texcoords.flip()
```

Comment:
Just to be sure: the kool `FloatBuffer` does not require flipping, whereas the one allocated by `BufferUtils` does?

Reply:
Cool, I just wanted to make sure these lines didn't get dropped by accident. :+1: